packages/builder/src/pages/builder/app/[application]/_components/BuilderSidePanel.svelte

```diff
@@ -366,18 +363,21 @@
     const payload = [
       {
         email: newUserEmail,
-        builder: !!creationRoleType === Constants.BudibaseRoles.Admin,
-        admin: !!creationRoleType === Constants.BudibaseRoles.Admin,
+        builder: { global: creationRoleType === Constants.BudibaseRoles.Admin },
+        admin: { global: creationRoleType === Constants.BudibaseRoles.Admin },
       },
     ]
 
-    if (creationAccessType === Constants.Roles.CREATOR) {
-      payload[0].appBuilders = [prodAppId]
-    } else {
-      payload[0].apps = {
-        [prodAppId]: creationAccessType,
+    if (creationRoleType !== Constants.BudibaseRoles.Admin) {
```

Comment:
can this be simplified? The if in an if, could it just be:
```
const notCreatingAdmin = creationRoleType !== Constants.BudibaseRoles.Admin
const isCreator = creationAccessType === Constants.Roles.CREATOR
if (notCreatingAdmin && isCreator) {
  payload[0].builder.apps = [prodAppId]
} else if (notCreatingAdmin && !isCreator) {
  payload[0].apps = {
    [prodAppId]: creationAccessType,
  }
}
```
Bit easier to understand what is happening here.